programmers/프린터.js

```diff
@@ -0,0 +1,15 @@
+function solution(priorities, location) {
+    let firstnum = 0;
+    let count = 0;
+    
+    while (priorities.length > 0) {
+        firstnum = priorities.shift();
+        if (firstnum < Math.max(...priorities)) priorities.push(firstnum);
+        else {
+              count += 1;
+              if (location === 0) return count;
+        }
+        location -= 1;
+        if (location === -1) location = priorities.length - 1;
+    }
+
```

Comment:
이 함수는 뭐를 return 하는거죠?
commit 해주실때 꼭 답인 answer와 answer가 return되고 있는지 확인 후에 올려주세요 !!
```suggestion
function solution(priorities, location) {
    const answer = [];
    let firstnum = 0;
    let count = 0;

    while (priorities.length > 0) {
        firstnum = priorities.shift();
        if (firstnum < Math.max(...priorities)) priorities.push(firstnum);
        else {
              count += 1;
              if (location === 0) return count;
        }
        location -= 1;
        if (location === -1) location = priorities.length - 1;
    }
    answer.push(count);
    
    return answer;
}
```

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ  아 뭔가 했네요  일부러 리턴을 안했어요   보시면 11번째줄 보시면  카운트만  리턴하면  되는 문제여서  굳이 또  리턴을  해줄 필요가 없더라구요   아무튼 피드백  감사해요